modules/k8s/tanka/new-app.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
```

Comment:
We should `shellcheck` this (there's quite a few suggestions)

Reply:
?

Reply:
ran through https://www.shellcheck.net/ and fixed warnings